app/javascript/lib/utils.ts

```diff
@@ -215,7 +213,12 @@ export const locales = [
 ];
 
 export function enumToOptions(en: Record<string, string>) {
-  return Object.entries(en).map(
-    ([key, value]) => ({ id: key, name: value })
-  );
+  return Object.entries(en).map(([key, value]) => ({ id: key, name: value }));
+}
+
+export function isObjectKey<T extends object>(
```

Comment:
export function isObjectKey(obj: any, key: PropertyKey): key is keyof typeof obj {
  return key in obj;
}